groovy/postInit/chains/Chemistry/VanadiumChain.groovy

```diff
@@ -0,0 +1,78 @@
+import gregtech.api.GregTechAPI;
+import gregtech.api.unification.material.Material;
+
+import static gregtech.api.unification.material.Materials.*;
+import static globals.Globals.*
+
+BR = recipemap('batch_reactor')
+ROASTER = recipemap('roaster')
+MIXER = recipemap('mixer')
+ELECTROLYZER = recipemap('electrolyzer')
+REACTION_FURNACE = recipemap('reaction_furnace')
+EBF = recipemap('electric_blast_furnace')
+
+REACTION_FURNACE.recipeBuilder()
+        .inputs(metaitem('dustVanadinite') * 21)
+        .inputs(metaitem('dustSalt') * 6)
+        .outputs(metaitem('dustSodiumVanadate') * 15)
+        .outputs(metaitem('dustLeadChloride') * 6)
+        .outputs(metaitem('dustLeadOxide') * 6)
+        .duration(200)
+        .EUt(120)
+        .buildAndRegister()
+
+REACTION_FURNACE.recipeBuilder()
+        .inputs(metaitem('dustVanadinite') * 42)
+        .inputs(metaitem('dustSodaAsh') * 18)
+        .outputs(metaitem('dustSodiumVanadate') * 30)
+        .outputs(metaitem('dustLeadChloride') * 3)
+        .outputs(metaitem('dustLeadOxide') * 18)
+        .fluidOutputs(fluid('carbon_dioxide') * 3000)
+        .duration(300)
+        .EUt(120)
+        .buildAndRegister()
+
+ELECTROLYZER.recipeBuilder()
+        .fluidInputs(fluid('lead_chloride') * 432)
+        .notConsumable(metaitem('graphite_electrode') * 2)
+        .outputs(metaitem('dustLead') * 1)
+        .fluidOutputs(fluid('chlorine') * 2000)
+        .duration(160)
+        .EUt(30)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .inputs(metaitem('dustSodiumVanadate') * 5)
+        .fluidInputs(fluid('water') * 1000)
+        .fluidOutputs(fluid('sodium_vanadate_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+BR.recipeBuilder()
+        .inputs(metaitem('dustAmmoniumChloride') * 6)
+        .fluidInputs(fluid('sodium_vanadate_solution') * 1000)
+        .outputs(metaitem('dustAmmoniumMetavanadate') * 9)
+        .fluidOutputs(fluid('salt_water') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+ROASTER.recipeBuilder()
+        .inputs(metaitem('dustAmmoniumMetavanadate') * 18)
+        .outputs(metaitem('dustVanadiumPentoxide') * 7)
+        .fluidOutputs(fluid('steam') * 1000)
+        .fluidOutputs(fluid('ammonia') * 2000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+EBF.recipeBuilder()
```

Comment:
Lead dioxide already has a processing route in LeadChain.groovy. Remove this recipe.